frontend/app/src/components/settings/data-security/data-management/PurgeData.vue

```diff
@@ -112,37 +119,49 @@ const { status, pending, showConfirmation } = useCacheClear<Purgeable>(
       </div>
     </div>
 
-    <VRow class="mb-0" align="center">
-      <VCol>
-        <VAutocomplete
-          v-model="source"
-          outlined
-          :label="t('purge_selector.label')"
-          :items="purgable"
-          item-text="text"
-          item-value="id"
-          :disabled="pending"
-          hide-details
-        />
-      </VCol>
-      <VCol cols="auto">
-        <VTooltip open-delay="400" top>
-          <template #activator="{ on, attrs }">
-            <VBtn
-              v-bind="attrs"
+    <div class="flex items-center gap-4">
+      <div class="flex flex-col md:flex-row md:gap-4 flex-1">
+        <div class="flex-1">
+          <VAutocomplete
+            v-model="source"
+            outlined
+            :label="t('purge_selector.label')"
+            :items="purgable"
+            item-text="text"
+            item-value="id"
+            :disabled="pending"
+          />
+        </div>
+        <div class="flex-1">
+          <LocationSelector
+            v-if="source === ALL_TRANSACTIONS"
+            v-model="evmChainToClear"
+            required
+            outlined
+            clearable
+            :items="txEvmChainsToLocation"
+            :label="t('purge_selector.evm_chain_to_clear.label')"
+            :hint="t('purge_selector.evm_chain_to_clear.hint')"
+          />
+        </div>
```

Comment:
here also we might be able to remove the wrapper divs and move the class to the items themselves